README.md

```diff
@@ -20,10 +20,13 @@ Please check [https://flang.dev](https://flang.dev) for language and implementat
 > Note that building from powershell/cmd does not work yet.
 
 1) Install chocolatey: https://chocolatey.org/install
-2) choco install git openjdk make msys2 diffutils
-3) in C:\tools\msys64\msys2_shell.cmd change line: 'rem set MSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
-4) execute: C:\tools\msys64\msys2_shell.cmd
-5) pacman -S mingw-w64-x86_64-clang
+2) In Powershell:
+2.1) choco install git openjdk make msys2 diffutils
+2.2) [Environment]::SetEnvironmentVariable("Path","c:\tools\msys64\mingw64\bin;" + $env:Path , "User")
+3) In File C:\tools\msys64\msys2_shell.cmd change line: 'rem setMSYS2_PATH_TYPE=inherit' to 'set MSYS2_PATH_TYPE=inherit'
```

Comment:
use lowercase in 'file'

Is this a typo: 'setMSYS2_PATH_TYPE', missing a space?